QoL pull request (#234)
myfreeweb/awesome-gtk: README.md

- [Monophony](https://gitlab.com/zehkira/monophony) - Application for streaming music from YouTube `#python` `#gtk4` `#libadwaita`.
- [Muzika](https://github.com/vixalien/muzika) - Music player with customizable home screen and Google Music integration `#gjs` `#gtk4` `#libadwaita`.
- [netease-cloud-music-gtk](https://github.com/gmg137/netease-cloud-music-gtk) - Audio player for the Netease Cloud Music `#rust` `#gtk4` `#libadwaita`.

> - [Recordbox](https://codeberg.org/edestcroix/Recordbox/) - A relatively simple, opinionated music player. Written in Rust so my inefficient code goes just that much faster.

Reviewer: ```suggestion
- [Recordbox](https://codeberg.org/edestcroix/Recordbox) - Music player designed primarily to be as simple as possible `#rust` `#libadwaita`.
```

- [HackUp](https://github.com/mdh34/hackup) - [Hacker News](https://news.ycombinator.com) client `#vala` `#gtk3` `#granite`.
- [Imaginer](https://github.com/ImaginerApp/Imaginer) - AI image generator (DALL·E 2, Portrait Plus, Stable Diffusion, Custom Provider) interface `#python` `#gtk4` `#libadwaita`.
- [Lobjur](https://github.com/ranfdev/Lobjur) - [lobste.rs](https://lobste.rs) client `#gjs` `#gtk4` `#libadwaita`.

> - [Newelle](https://github.com/qwersyk/Newelle) - Virtual assistant (based on GPT4All) that interacts with the system by running shell commands and execute Python code `#python` `#gtk4` `#libadwaita`.

Reviewer: I don't think this is the right category for Newelle.

Author: I think Bavarder is alternative of Newelle, and it placed in this

Reviewer: Bavarder is a font-end to chat bot services while Newelle provides more that a wrapper. In my opinion, Newelle deserves to be in its own category.

- [Paper Clip](https://apps.gnome.org/PdfMetadataEditor) - PDF metadata editor `#vala` `#gtk4` `#libadwaita` `#gnome`.
- [PDF Arranger](https://github.com/pdfarranger/pdfarranger) - PDF editor with merging, splitting, rotating, cropping and rearranging based on [pikepdf](https://github.com/pikepdf/pikepdf) `#python` `#gtk3` `#libhandy`.

> - [PDF Slicer](https://junrrein.github.io/pdfslicer) - Simple application to extract, merge, rotate and reorder pages of PDF documents with undo/redo support `#c++` `#gtk3`.

Reviewer: Why should this application be removed?

Author: 4 years without any updates

Reviewer: I guess we can remove it since PDF Arranger provides the same functionnality and is up to date.

- [Szyszka](https://github.com/qarmin/szyszka) - Fast bulk file renamer `#rust` `#gtk4`.
- [Ticket Booth](https://github.com/aleiepure/ticketbooth) - Application to keep track of TV series/movies with TMDB's API `#python` `#gtk4` `#libadwaita`.
- [TV Series Renamer](https://github.com/mmstick/tv-renamer) - TV series renaming application that support adding titles to episodes `#rust` `#gtk3`.

> - [Workspaces](https://github.com/DevAlien/workspaces) - Desktop workpaces for elementaryOS `#vala` `#gtk3` `#granite`.

Reviewer: Why should this application be removed?

Reviewer: Even if the code hasn't been updated for a while, there is no alternative for this feature on the list.

Author: Because it no longer makes sense, the multitasking view feature in Pantheon closes all user needs from this application and is implemented at the shell level

- [Anjuta](https://gitlab.gnome.org/Archive/anjuta) - (archived) IDE with a GUI designer for the GNOME desktop `#c` `#gtk2` `#gnome`.
- [GNOME Builder](https://apps.gnome.org/Builder) - Tool to help you write and contribute to great GNOME-based applications `#c` `#gtk4` `#libadwaita` `#gnome`.

> - [GtkIDE.jl](https://github.com/jonathanBieler/GtkIDE.jl) - GTK-based IDE for Julia `#julia` `#gtk3`.

Reviewer: Same here, I don't think this should be removed as there isn't an alternative Julia IDE on the list.

Author: Despite Jonathan's support for GTK bindings for julia, it's clear that this project of his is stagnating

- [Convolution](https://gitlab.gnome.org/bazylevnik0/convolution) - Logic game about escaping from a maze `#gjs` `#gtk4` `#libadwaita`.
- [Crosswords](https://gitlab.gnome.org/jrb/crosswords) - Game of crosswords with squpport for shaped and colors crosswords and `.ipuz`, `.jpuz`, `.xd`, and `.puz` files `#c` `#gtk4` `#libadwaita`.

> - [Flood It!](https://github.com/tfuxu/floodit) - Flood It is a game with the simple premise of flooding the entire board with one color in the least amount of moves possible.

Reviewer: ```suggestion
- [Flood It!](https://github.com/tfuxu/floodit) - Simple game with the premise of flooding the entire board with one color in the least amount of moves possible `#go` `#libadwaita`.
```
